### ISSUE TITLE
Update with empty fields returns an SQL error

### ISSUE THREAD

**Issue by robinjmurphy** [robconery/massive-js#166]
When running an update without specifying any fields, e.g.

```js
db.people.update({ id: 2 }, cb);
```

Or, more explicitly:

```js
const conditions = { id: 2 };
const fields = {};

db.people.update(conditions, fields, cb);
```

An SQL error is returned:

```
syntax error at or near "WHERE"
```

This is because there are no `SET` statements in the [resulting SQL query](https://github.com/robconery/massive-js/blob/eed8887634b57fce120add45c4e0e911ce5351af/lib/table.js#L72).

It's a bit of a strange query to be running, but I'm creating an API that supports PATCH requests and it's possible that a client could send an empty object.

I'd like to help fix this if it's something that would be useful. Either by extending the [existing `fields` check](https://github.com/robconery/massive-js/blob/eed8887634b57fce120add45c4e0e911ce5351af/lib/table.js#L61) to make sure that there are actually some keys in the object or just running a `SELECT` query instead.

Let me know if it's something you'd welcome and I can create a PR.

**Comment by robconery** [comment]
A PR would be great - thanks :).

**Comment by robinjmurphy** [comment]
Fixed in #167
